pupil_src/launchables/world.py

```diff
@@ -100,12 +100,13 @@ def start_stop_eye(eye_id, make_alive):
         else:
             stop_eye_process(eye_id)
 
-    def detection_enabled_getter() -> bool:
-        return g_pool.pupil_detection_enabled
+    def detection_enabled_getter() -> int:
+        return int(g_pool.pupil_detection_enabled)
```

Comment:
Why not convert to bool directly? That would fit better to the `detection_enabled_setter` call signature
```suggestion
    def detection_enabled_getter() -> bool:
        return bool(g_pool.pupil_detection_enabled)
```

Reply:
The `selected` property of `ui.Switch` is of type `bint` ([see here](https://github.com/pupil-labs/pyglui/blob/3d52f8ed9ad7d6cdc7dab08a2b7bd5aa3f454df5/pyglui/ui_elements.pxi#L238)). These type annotations are more for documentation purposes, and to keep in mind to do the `int` to `bool` conversion in the setter before sending the notification.